pytripgui/main.py

```diff
@@ -230,7 +230,7 @@ def view_about(self, evt):
         info.SetName('PyTRiPGUI')
         info.SetVersion(pytripgui_version)
         info.SetDescription(description)
-        info.SetCopyright('(C) 2012 - 2016 Aarhus Particle Therapy Group')
+        info.SetCopyright('(C) 2012 - 2017 PyTRiP98 Developers')
```

Comment:
My attempt was here:
https://github.com/pytrip/pytripgui/pull/88/files#diff-5a3aa1c8d3bfe0d81a3fcc9f4da31b17

there will be a conflict